starter-code/functions-and-arrays.js

```diff
@@ -1,16 +1,26 @@
 // Find the maximum
 function maxOfTwoNumbers (first, second) {
-
+  if(first > second) {
+    return first;
+  } else {
+    return second;
+}
 }
-
 var largest = maxOfTwoNumbers(2, 6);
 console.log(largest);
 
 // Finding Longest Word
 function findLongestWord (words) {
-
+  var wordstore= words[0];
+  for(i=0; i<words.length;i+=1) {
+  if (wordstore.length < words[i].length) {
+    wordstore = words[i];
+    console.log("wordstore for greater loop: " + wordstore );
+  } else {
+    console.log("wordstore for else loop: " + wordstore );
+  }}
+return wordstore;
 }
```

Comment:
your solution is ok but you could try this with a forEach loop:

function findLongestWord (words) {
  var longest = "";

  words.forEach (function (word) {
    if (word.length > longest.length) {
      longest = word;
    }
  });

  return longest;
}